### PR TITLE
Update macOS build for 2.15

### PR DESCRIPTION
**Pull request by psiinon**

### CODE REVIEW

File: buildSrc/src/main/java/org/zaproxy/zap/tasks/CreateDmg.java [hunk 1]
```diff
@@ -56,7 +56,7 @@ void create() {
                                             "-format",
                                             "UDBZ",
                                             "-megabytes",
-                                            "800",
+                                            "2000", // Too low fails, but bigger doesnt seem to affect the dmg size
```
[the remark this answers is not in the document]
Comment (kingthorin) [reply buildSrc/src/main/java/org/zaproxy/zap/tasks/CreateDmg.java:59]: There's no like "size as needed" option?

**Review by kingthorin — APPROVED** [review]

File: buildSrc/src/main/java/org/zaproxy/zap/tasks/CreateDmg.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (thc202) [reply buildSrc/src/main/java/org/zaproxy/zap/tasks/CreateDmg.java:59]: Maybe by specifying a type? https://ss64.com/mac/hdiutil.html

[the remark this answers is not in the document]
Comment (kingthorin) [reply buildSrc/src/main/java/org/zaproxy/zap/tasks/CreateDmg.java:59]: Ya it sounds like it:
> UDIF is the default type.  If specified, a UDRW of
 the specified size will be created.  SPARSE creates
 a UDSP: a read/write single-file image which expands
 as _it_ is filled with data.  SPARSEBUNDLE creates a
 UDSB: a read/write image backed by a directory bun-
 dle.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**